src/components/v2v3/V2V3Project/ProjectDashboard/hooks/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from './useAboutPanel'
```

Comment:
Is it possible to remove this barrel file? We are trying to clean up all usage of barrel files where we can.

I think @aeolianeth recently nuked all barrels from the project

https://dev.to/tassiofront/barrel-files-and-why-you-should-stop-using-them-now-bc4

Reply:
All good - lets leave it for now! Thanks for the callout though @wraeth-eth 🙏